src/picongpu/include/fields/laserProfiles/laserPolynom.hpp

```diff
@@ -59,7 +59,7 @@ HINLINE float3_X laserLongitudinal(uint32_t currentStep, float_X& phase)
     elong.x() = float_X(
                        float_64(AMPLITUDE) *
```

Comment:
Please open an issue for that. 
Conversions are out of the scope of this pull request.